arviz/plots/plot_utils.py

```diff
@@ -602,6 +603,23 @@ def color_from_dim(dataarray, dim_name):
     return colors, color_mapping
 
 
+def vectorized_to_hex(rgba_c):
+    """Converts a vector of RGBA values to a vector of hex values
+
+    Parameters
+    ----------
+    rgba_c: vector of RGBA values
+
+    Returns
+    -------
+    rgba_hex : vector of hex values
+    """
+    rgba_hex = []
+    for rgba in range(len(rgba_c)):
+        rgba_hex.append(to_hex(rgba_c[rgba]))
+    return rgba_hex
```

Comment:
I think this could be a list comprehension, it's better to avoid lists growing inside of loops if possible.

The keep_alpha (not sure the name is exactly this one, I hope the idea is clear) should also be explicit and passed to to hex, we may want to keep or not the alpha value in the resulting hex color.

Reply:
For that, should I add a keep_alpha parameter in the main plot function too?

Reply:
No, the default is to set different alpha values to the colors themselves, so that the relevant points (large khat) are opaque whereas low khat is quite transparent. This can already be changed using plot_kwargs, there is no need for a new keyword argument.